src/unit_tests/wrappers/wazuh/syscheckd/fim_db_wrappers.c

```diff
@@ -190,3 +192,33 @@ int __wrap_fim_db_sync_path_range(fdb_t *fim_sql,
 
     return mock();
 }
+
+/* Auxiliar functions */
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of fim_db_get_path_range
+*/
+void expect_fim_db_get_path_range_call(const fdb_t *db,
+                                           const char *start_str,
+                                           const char *top_str,
+                                           int storage,
+                                           fim_tmp_file *tmp_file,
+                                           int ret) {
+
+    expect_value(__wrap_fim_db_get_path_range, fim_sql, db);
+    expect_value(__wrap_fim_db_get_path_range, type, FIM_TYPE_FILE);
+    expect_string(__wrap_fim_db_get_path_range, start, start_str);
+    expect_string(__wrap_fim_db_get_path_range, top, top_str);
+    expect_value(__wrap_fim_db_get_path_range, storage, storage);
+    will_return(__wrap_fim_db_get_path_range, tmp_file);
+    will_return(__wrap_fim_db_get_path_range, ret);
+}
+
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of fim_db_delete_range
```

Comment:
This function **loads**. Also, this doxygen comment should be in the _.h_ file instead.